<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="nickLabel">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nickEdit">
          <property name="text">
           <string>NoName</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="connectLabel">
          <property name="text">
           <string>Server Ip:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="connectEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="chatOutput">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="messageBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>91</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectToServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectToServer()</slot>
  <slot>sendMessage()</slot>
 </slots>
</ui>
